.github/dependabot.yml

```diff
@@ -0,0 +1,41 @@
+# Basic dependabot.yml file with
+# minimum configuration for two package managers
+
+version: 2
+updates:
+  # Enable version updates for python
+  - package-ecosystem: "pip"
+    # Look for a `requirements` in the `root` directory
+    directory: "/"
+    # Check for updates once a week
+    schedule:
+      interval: "monthly"
+    # Labels on pull requests for version updates only
+    labels:
+      - "dependencies"
+    pull-request-branch-name:
+      # Separate sections of the branch name with a hyphen
+      # for example, `dependabot-npm_and_yarn-next_js-acorn-6.4.1`
+      separator: "-"
+    # Allow up to 5 open pull requests for pip dependencies
+    open-pull-requests-limit: 3
+
+  # Enable version updates for GitHub Actions
```

Comment:
it is not clear about what will it update?

Reply:
It will update the version like here:
 https://github.com/Lightning-AI/LitServe/blob/a7b5910d0479f1b034ab927c7cad3029c9f3d101/.github/workflows/ci-testing.yml#L61
where it shall be already `v5`